sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -435,16 +435,16 @@ def make_request(query):
 
 
 def clear_bucket():
```

Comment:
Please regularly merge main branch into the refactoring branch, because Laurentiu keeps working on the DAGs